eclcli/common/utils.py

```diff
@@ -347,6 +347,20 @@ def validate_ipv4(text):
         raise exceptions.CommandError(msg % text)
     return text
 
+def validate_int_range(text, min_val, max_val):
+    try:
+        if not isinstance(text, int):
```

Comment:
I think this if-else block is not needed. I prefer simply text gets casted to int.